scripts/templates/build_meta.py

```
@@ -0,0 +1,92 @@
+        help="표준 카테고리 스키마 JSON 경로",
+    )
+    parser.add_argument("--slides-dir", type=Path, help="슬라이드 JPG 출력 디렉터리")
+    parser.add_argument("--thumbnail", type=Path, help="그리드 thumbnail.jpg 출력 경로")
+    parser.add_argument("--text-output", type=Path, help="임시 텍스트 markdown 출력 경로")
+    parser.add_argument("--model", help="LLM 모델명 override")
+    parser.add_argument("--temperature", type=float, default=0.1, help="LLM temperature")
+    parser.add_argument("--soffice-bin", default="soffice", help="LibreOffice 실행 파일")
+    parser.add_argument("--pdftoppm-bin", default="pdftoppm", help="pdftoppm 실행 파일")
+    parser.add_argument("--timeout-seconds", type=float, default=60.0, help="외부 명령 타임아웃")
+    parser.add_argument("--dpi", type=int, default=150, help="슬라이드 JPG 렌더링 DPI")
+    parser.add_argument("--tmp-root", type=Path, default=Path("/tmp"), help="렌더링 임시 디렉터리")
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)